docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md

```diff
@@ -0,0 +1,772 @@
+TCK Results
+===========
+
+
+As required by the
+[Eclipse Foundation Technology Compatibility Kit License](https://www.eclipse.org/legal/tck.php),
+following is a summary of the TCK results for releases of Jakarta EE Platform 10.0, certification summary.
+
+# Jakarta EE Platform 10.0, Eclipse GlassFish 7.0, TCK Certification Summary
+
+- Product Name, Version and download URL (if applicable): <br/>
+  [Eclipse GlassFish Downloads](/glassfish/download)<br/>
+  [glassfish-7.0.zip](https://download.eclipse.org/ee4j/glassfish/glassfish-7.0.0.zip)
+
+- Specification Name, Version and download URL: <br/>
+  [Jakarta EE Platform, 10.0](https://jakarta.ee/specifications/platform/10/)
+
+- TCK Version, digital SHA-256 fingerprint and download URL: <br/>
+  [Jakarta EE Platform TCK 10.0](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip), 
+  SHA-256: `930c1b5037cd60fb1f59c6473ad49170337fb3423c1d56bf34f9dcc1907ed06f`
+
+- Public URL of TCK Results Summary: <br/>
+  [TCK results summary](./TCK-Results-7.0)
+  
+- Any Additional Specification Certification Requirements: <br/>
+  Jakarta Concurrency 3.0 TCK <br/>
+  Jakarta JSON Processing 2.1 TCK <br/>
+  Jakarta JSON Binding 3.0 TCK <br/>
+  Jakarta RESTful Web Services 3.1 TCK <br/>
+  Jakarta Security 3.0 TCK <br/>
+  Jakarta Authentication 3.0 TCK <br/>
+  Jakarta Faces 4.0 TCK <br/>
+  Jakarta Contexts and Dependency Injection 4.0 TCK <br/>
```

Comment:
I'm trying to better understand the requirement for also passing the CDI 4.0 Language Model TCK.  Related to that is https://github.com/jakartaee/cdi-tck/issues/391

Reply:
All clear now, passing CDI should include running the [CDI Language Model TCK](https://github.com/jakartaee/cdi-tck/blob/master/doc/reference/src/main/asciidoc/executing.asciidoc#executing-the-lang-model-test-suite).